test/src/kava.spec.js

```diff
@@ -126,6 +126,47 @@ describe('KavaPlugin', function() {
       type: 'Vod',
       dvr: false
     };
+    const badconfig = {
```

Comment:
Please delete the non relevant config from bad config just to be more clear about why does it break.
session, playback, kava plugin

Reply:
done